app/repository/hbase/legalunit/LegalUnitRowMapper.scala

```diff
@@ -0,0 +1,39 @@
+package repository.hbase.legalunit
+
+import scala.util.Try
+
+import uk.gov.ons.sbr.models.enterprise.{ EnterpriseLink, Ern }
+import uk.gov.ons.sbr.models.legalunit.{ LegalUnit, UBRN }
+
+import repository.RestRepository.Row
+import repository.RowMapper
+import repository.hbase.legalunit.LegalUnitColumns._
+
+/*
+ * The following fields are optional:
+ * - luref
+ * - entref
+ * - tradingstyle
+ * - address2
+ * - address3
+ * - address4
+ * - address5
+ *
```

Comment:
These should be declared in your LegalUnitsColumns

Reply:
luref is local unit ref so remove from the comment